src/Header.svelte

```diff
@@ -227,6 +228,7 @@
     <div class="toggle" on:click={toggleNavbar}>
       <Icon name="ellipsis" width={27} height={27} fill />
     </div>
+    <ThemeToggle />
```

Comment:
We should be able to pass the current theme to `ThemeToggle` as a prop e.g. `<ThemeToggle theme={$session.theme} />`

Reply:
2b1adbfae38338e4f7bd93457e16f40ab2eb8ae5

Reply:
On a second thought. I think it doesn't make sense to mix the appearance state with the wallet state that we call "session". The two are completely independent and should be in separate modules. In my opinion it's best to keep the appearance state in a module within the theme component, this way it's all in one place and the state can still be imported by other modules if needed.